<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrajectoriesDialog</class>
 <widget class="QDialog" name="TrajectoriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trajectories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>50</number>
     </attribute>
     <column>
      <property name="text">
       <string>Visible</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="propertiesBox">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="colorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="thicknessLabel">
        <property name="text">
         <string>Thickness</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="thicknessSpin">
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="startTimeLabel">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="startTimeSpin">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="endTimeLabel">
        <property name="text">
         <string>End Time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="endTimeSpin">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
